src/index.html

```diff
@@ -3,11 +3,294 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBiKE</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <link
+      rel="icon"
+      type="image/svg+xml"
+      href="favicon.svg"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header" id="home">
+      <nav class="navigation header__navigation">
+        <a href="#" class="logo navigation__logo">
+          <img
+          class="logo__image"
+          src="./images/logo-mobile.svg"
+          alt="MyBiKE logo mobile"
+          >
+        </a>
+
+        <a href="tel:+1 234 555-55-55" class="phone navigation__item">
+          <span class="phone__label">+1 234 555-55-55</span>
+
+          <img
+          class="phone__icon"
+          src="./images/phone.svg"
+          alt="Phone logo"
```

Comment:
Class is icon, alt says logo, try to be consistent. Also some of your alt descriptions are not meaningful enough, alt attribute should describe what the picture represents to for example blind person that uses screen reader